tests/Neo.UnitTests/Network/P2P/Payloads/UT_Signers.cs

```diff
@@ -23,6 +23,51 @@ namespace Neo.UnitTests.Network.P2P.Payloads
     [TestClass]
     public class UT_Signers
     {
+        [TestMethod]
+        public void Test_IEquatable()
+        {
+            var expected = new Signer()
+            {
+                Account = UInt160.Zero,
+                Scopes = WitnessScope.Global,
+                AllowedContracts = [],
+                AllowedGroups = [],
+                Rules = []
+            };
+
+            var actual = new Signer()
+            {
+                Account = UInt160.Zero,
+                Scopes = WitnessScope.Global,
+                AllowedContracts = [],
+                AllowedGroups = [],
+                Rules = []
+            };
+
+            var notEqual = new Signer()
+            {
+                Account = UInt160.Zero,
+                Scopes = WitnessScope.WitnessRules,
+                AllowedContracts = [],
```

Comment:
We should add some contracts and groups for ensure that is well checked

Reply:
Ill add them later in other PR. I think `ECPoint` and `UInt160` don't inherit `IEquatable`

Reply:
done!